iothub_client/src/iothubtransport_amqp_common.c

```diff
@@ -2007,6 +2007,18 @@ IOTHUB_CLIENT_RESULT IoTHubTransport_AMQP_Common_SetOption(TRANSPORT_LL_HANDLE h
                 result = IOTHUB_CLIENT_OK;
             }
         }
+        else if (strcmp(OPTION_RETRY_INTERVAL_SEC, option) == 0)
+        {
+            if (retry_control_set_option(transport_instance->connection_retry_control, RETRY_CONTROL_OPTION_INITIAL_WAIT_TIME_IN_SECS, value) != 0)
```

Comment:
wait is this defined in the repo elsewhere and it just wasn't being used?

Reply:
Yes, this was never called so we weren't able to ever set it.

---
In reply to: [323444610](https://github.com/Azure/azure-iot-sdk-c/pull/1172#discussion_r323444610) [](ancestors = 323444610)